Addresses issue 312

kgonzago opened the pull request:
This fix addresses #312. It allows:
1. A user to supply unequally spaced rings
2. Checks if number of rings supplied and unequally spaced ring values are the same

csmoore commented on source/addins/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/ViewModels/RangeViewModel.cs:
```diff
@@ -176,7 +243,7 @@ public override bool CanCreateElement
                     return (Point1 != null && NumberOfRadials >=0);
                 else
-                    return (Point1 != null && NumberOfRings > 0 && NumberOfRadials >= 0 && Distance > 0.0);
+                    return (Point1 != null && NumberOfRings > 0 && NumberOfRadials >= 0 && RangeIntervals.Count > 0 /*Distance > 0.0*/);
```
Some inline commented out code

csmoore commented on source/addins/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/ViewModels/RangeViewModel.cs:
```diff
@@ -270,7 +337,7 @@ private IGeometry DrawRings()
                 {
                     // set the current radius
-                    radius += Distance;
+                    radius += RangeIntervals.Count == 1 ? RangeIntervals[0] : RangeIntervals[x]; //Distance;
```
commented code

kgonzago commented:
@csmoore made changes you suggested:
1. removed commented out code
2. set default `Distance` value to 1.0
3. added validation check of 0 length for ring intervals
